camelot/model/memento.py

```diff
@@ -82,6 +81,10 @@ class Memento( Entity ):
                                   nullable = False,
                                   index = True )    
     previous_attributes = orm.deferred( schema.Column( PickleType() ) )
+
+    __facade_args__ = {
```

Comment:
dit was idd wat ik in gedachten had.

ik bedacht net dat het nog beter zou zijn indien we op niveau van de mapper konden instellen dat een bepaalde class
niet wijzigbaar is via de ORM en dat updates via de ORM dan zouden falen.

maar ik kan zo niks vinden op het 1e zicht, dus di een goede werkhypothese denk ik